style.css

```diff
@@ -0,0 +1,758 @@
+    line-height: 17px;
+    color: #515151;
+    background: transparent;
+    border: 8.49673px solid #fff;
+  }
+
+  .seminar-schedule {
+    color: #595959;
+    font-family: 'Lato', sans-serif;
+    font-style: normal;
+    font-weight: 900;
+    font-size: 1.25rem;
+    padding-left: 2%;
+    padding-top: 5.4%;
+    margin: 0;
+  }
+
+  .venue {
+    color: #868686;
+    font-family: 'Lato', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 0.815rem;
+    padding-top: 2%;
+    padding-left: 2%;
+    padding-bottom: 5%;
+  }
+
+  /* Main program starts */
+
+  .main-program {
+    background-image: url(./assets/images/program-bg.png);
+  }
+
+  .main-program-header {
+    color: #fff;
+    text-align: center;
+  }
+
+  .main-border {
+    width: 2rem;
+    border-bottom: 1px solid rgb(239, 81, 60);
+    margin: 0 auto 4rem;
+  }
+
+  .main-program-header h2 {
+    font-weight: 400;
+    font-size: 1.27rem;
+    margin: 4% 0%;
+    border-bottom: 2px solid red;
+    display: inline-block;
+  }
+
+  .program-content {
+    display: flex;
+    gap: 15px;
+    align-items: center;
+    background: rgba(255, 255, 255, 0.1);
+    margin: 0 3% 2% 3%;
+  }
+
+  .image-wrap {
+    display: flex;
+  }
+
+  .program {
+    width: 31%;
+    display: flex;
+    justify-content: center;
+    text-align: center;
+    color: #ff5438;
+    font-weight: 700;
+    font-size: 3vw;
+  }
+
+  .program-content img {
+    margin: 15% 0 15% 15%;
+  }
+
+  .program-content p {
+    color: #fff;
+    font-weight: 500;
+    font-size: 2.5vw;
+    letter-spacing: 0.06em;
+  }
+
+  .program-btn {
+    align-self: center;
+    text-align: center;
+    padding: 8% 0 10.5%;
+  }
+
+  .join {
+    background: #ff4f38;
+    color: #fff;
+    font-size: 5vw;
+    font-weight: 400;
+    padding: 2%;
+  }
+
+  .see-program {
+    display: none;
+  }
+
+  /* Main program ends */
+
+  /* Featured Speaker Starts */
+
+  .speaker-intro {
+    margin-bottom: 10%;
+    text-align: center;
+    padding-top: 4.18%;
+  }
+
+  .speaker-intro-border {
+    width: 2rem;
+    border-bottom: 1px solid #a04038;
+    margin: 0 auto 2.5rem;
+  }
+
+  .speaker {
+    display: flex;
+    margin: 0 0 6% 3%;
+  }
+
+  .speaker img {
+    max-width: 35%;
+    max-height: 5%;
+  }
+
+  .speaker-details {
+    padding-left: 4.2%;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+  }
+
+  .speaker h3 {
+    color: #000;
+    font-weight: 700;
+    font-size: 0.95rem;
+  }
+
+  .designition {
+    color: #f78c89;
+    font-weight: 700;
+    font-size: 0.63rem;
+    margin: 0;
+    margin-bottom: 1.8%;
+  }
+
+  .speaker-border {
+    width: 2rem;
+    border-bottom: 1px solid #d9d9d9;
+  }
+
+  .speaker-work {
+    color: #000;
+    font-weight: 700;
+    font-size: 0.6rem;
+    padding-top: 0.8rem;
+  }
+
+  /* Rights Starts */
+  .rigts {
+    display: flex;
+    justify-content: space-between;
+    border-top: 1px solid #3b3e4544;
+  }
```

Comment:
- [OPTIONAL] Kindly make sure the `footer`  on the `index` mobile screen is not available in the Behance design.

   - ### YOUR DESIGN
   
     ![image](https://github.com/Zafron047/Capstone_Project/assets/87197412/f18192f2-3108-4417-bae5-854513b78e58)

   - ### BEHANCE DESIGN
   
   ![image](https://github.com/Zafron047/Capstone_Project/assets/87197412/29d540db-3250-4bdf-b2aa-ea92901cdefd)